R/plots.R

```diff
@@ -348,6 +354,7 @@ transformDates <- function(plotsols){
   # Convert to POSIXct if feasible. This will require some thought to figure out
   # which sorts of Period formats can be generated by predLoad and predConc. In
   # the meantime, here we address just a couple of the possibilities.
+  is.POSIXt=DATE=is.Date=NULL
```

Comment:
definitely not. `is.POSIXt` and `is.Date` are functions that need to be declared with an @importFrom, not by setting them to NULL, this function won't work as changed. DATE should be set like this, please:
```{r}
DATE <- '.transform.var'
```